styleguide/_themes/derek/scss/snowflakes/homepage.scss

```diff
@@ -92,16 +92,16 @@
 }
 
 .hostingOptions-logo {
-  @include make-sm-column(5);
-  @include make-sm-column-push(3.5);
+  @include make-sm-column(3.5);
```

Comment:
ehh....  I guess it's fine for now, since we're not looking to refactor everything.  But wanted to clarify that I don't necessarily like it and we shouldn't try hard to avoid doing this moving forward.